external/googleapis/renovate.sh

```diff
@@ -14,8 +14,17 @@
 # limitations under the License.
 
 set -euo pipefail
+BOLD_ON_GREY=$(tput bold && tput setab 252)
+TURN_OFF_ATTR=$(tput sgr0)
+SEPARATOR=
 
-# Determine the googleapis HEAD commit and the checksum of its tarball.
+function banner() {
+  test -n "${SEPARATOR}" && echo
+  echo "${BOLD_ON_GREY}[ $* ]${TURN_OFF_ATTR}"
+  SEPARATOR=Y
+}
```

Comment:
I will be pleasantly surprised when I run the script and see what the output looks like. :smile:

Reply:
Yes (he says mesmerizingly), you will.